index.bs

```diff
@@ -1014,7 +1014,7 @@ optionally evidence of [=user consent=] to a specific transaction.
 
     :   <dfn>extensions</dfn>
     ::  This member contains additional parameters requesting additional processing by the client and authenticator. For example,
-        the caller may request that only authenticators with certain capabilies be used to create the credential, or that
+        the caller MAY request that only authenticators with certain capabilies be used to create the credential, or that
```

Comment:
This is descriptive - not normative language. Please back this individual change out.

Reply:
updated.